ha/core/system_health/system_health_manager.py

```diff
@@ -26,14 +26,15 @@ def __init__(self, store):
         """
         self._store = store
 
-    def get_key(self, key: str):
+    def get_key(self, key: str, just_value: str=True):
         """
         Get key method.
         """
         key_val = self._store.get(key)
-        if key_val is not None:
-            _, value = key_val.popitem()
-            return value
+        if just_value:
+            if key_val is not None:
```

Comment:
Condition 'key_val' is None and 'just_value' is True needs to be handled 
I saw that at the moment 'just_value' is set to false always, but it can change later and the condition might remain unhandled

Reply:
I think it is handled already, we would return "None".